website/search_migration/migrate.py

```diff
@@ -5,29 +5,32 @@
 
 import logging
 
+from django.utils import timezone
 from elasticsearch import helpers
 from modularodm.query.querydialect import DefaultQueryDialect as Q
 
+import website.search.search as search
+from framework.auth import User
 from framework.mongo.utils import paginated
+from scripts import utils as script_utils
 from website import settings
-from framework.auth import User
-from website.models import Node
 from website.app import init_app
-import website.search.search as search
-from scripts import utils as script_utils
+from website.institutions.model import Institution
+from website.models import Node
 from website.search.elastic_search import client as es_client
 from website.search.search import update_institution
-from website.institutions.model import Institution
 
 logger = logging.getLogger(__name__)
 
-def migrate_nodes(index):
+def migrate_nodes(index, query=None):
     logger.info('Migrating nodes to index: {}'.format(index))
-    query = Q('is_public', 'eq', True) & Q('is_deleted', 'eq', False)
-    total = Node.find(query).count()
+    node_query = Q('is_public', 'eq', True) & Q('is_deleted', 'eq', False)
+    if query:
+        node_query = query & node_query
+    total = Node.find(node_query).count()
     increment = 200
     total_pages = (total // increment) + 1
-    pages = paginated(Node, query=query, increment=increment, each=False)
+    pages = paginated(Node, query=node_query, increment=increment, each=False)
```

Comment:
I wonder if we can do a `.include('contributor__user__guids')`. I'll play around to see if it will work.